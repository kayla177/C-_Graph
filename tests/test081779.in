ENTITY BDJBYSRR name_BDJB institution
ENTITY DTLOQPU3 name_DTLO journal
ENTITY GI1RWG4C name_GI1R author
ENTITY 24WWEW6X name_24WW conference
ENTITY 0H9YC3R0 name_0H9Y paper
ENTITY J9RMT92H name_J9RM paper
ENTITY B7BHOYK4 name_B7BH conference
ENTITY 01Q4Y7KX name_01Q4 institution
ENTITY 3OYA78SV name_3OYA paper
ENTITY WKP8BS5S name_WKP8 author
RELATIONSHIP GI1RWG4C cited_by 3OYA78SV 8.4
RELATIONSHIP B7BHOYK4 cited_by J9RMT92H 4.9
RELATIONSHIP 24WWEW6X presented_at DTLOQPU3 3.7
RELATIONSHIP 0H9YC3R0 authored_by DTLOQPU3 0.4
RELATIONSHIP BDJBYSRR cited_by 0H9YC3R0 7.3
RELATIONSHIP 01Q4Y7KX collaborated_with DTLOQPU3 4.0
RELATIONSHIP WKP8BS5S presented_at B7BHOYK4 8.4
RELATIONSHIP 01Q4Y7KX cited_by B7BHOYK4 0.5
RELATIONSHIP WKP8BS5S cited_by 3OYA78SV 6.5
RELATIONSHIP 3OYA78SV cited_by DTLOQPU3 7.8
RELATIONSHIP 24WWEW6X cited_by 3OYA78SV 6.3
RELATIONSHIP WKP8BS5S collaborated_with DTLOQPU3 0.3
RELATIONSHIP J9RMT92H authored_by 3OYA78SV 7.9
RELATIONSHIP BDJBYSRR authored_by 24WWEW6X 7.0
RELATIONSHIP 01Q4Y7KX collaborated_with 0H9YC3R0 5.0
DELETE 24WWEW6X
PATH WKP8BS5S B7BHOYK4
PRINT GI1RWG4C
PATH WKP8BS5S J9RMT92H
PATH J9RMT92H DTLOQPU3
PRINT 0H9YC3R0
PRINT J9RMT92H
PRINT 0H9YC3R0
PRINT 3OYA78SV
PRINT J9RMT92H
EXIT
