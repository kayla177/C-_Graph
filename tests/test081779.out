success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
WKP8BS5S B7BHOYK4 8.4
3OYA78SV
WKP8BS5S 3OYA78SV J9RMT92H 14.4
J9RMT92H 3OYA78SV DTLOQPU3 15.7
DTLOQPU3 BDJBYSRR 01Q4Y7KX
B7BHOYK4 3OYA78SV
DTLOQPU3 BDJBYSRR 01Q4Y7KX
GI1RWG4C WKP8BS5S DTLOQPU3 J9RMT92H
B7BHOYK4 3OYA78SV
